samples/osgi-bundles/org.wso2.carbon.uuf.sample.simple-auth.bundle/pom.xml

```diff
@@ -61,8 +61,8 @@
                             org.wso2.carbon.kernel.utils; version="${carbon.kernel.version.range}",
                             org.wso2.carbon.kernel.context; version="${carbon.kernel.version.range}",
                             org.wso2.carbon.messaging; version="${carbon.messaging.version.range}",
-                            org.wso2.carbon.uuf.api.auth.*; version="${carbon.uuf.version}",
-                            org.wso2.carbon.uuf.spi.auth.*; version="${carbon.uuf.version}",
+                            org.wso2.carbon.uuf.api.*; version="${carbon.uuf.version}",
```

Comment:
org.wso2.carbon.uuf.spi.* is required since the PersistantSessionManager requires the `HTTPRequest` interface.